pkgs/development/misc/resholve/default.nix

```diff
@@ -16,13 +16,11 @@ let
     self = python27';
     pkgsBuildHost = pkgsBuildHost // { python27 = python27'; };
     # strip down that python version as much as possible
-    openssl = null;
```

Comment:
I added this back since I thought "why would this be required?", and was able to build resholve + a script that uses it just fine.

Should I PR, or is there something I'm missing? (I suspect you removed it to test, but never tested with just libffi removed.)

Reply:
Yeah please.

Reply:
I remember running into issues with oil tests, but it seems to be working now, so I'm not sure what really happened